trunk/SUAVE/Analyses/Stability/AVL.py

```diff
@@ -147,12 +137,10 @@ def initialize(self,conditions):
         mac                 = geometry.wings['main_wing'].chords.mean_aerodynamic        
         mach                = conditions.freestream.mach_number
         AoA                 = conditions.aerodynamics.angle_of_attack
```

Comment:
I actually think it looks better with the spacing in this section. Some of these assignments can be logically separated.